src/Type/Definition/ListOfType.php

```diff
@@ -30,9 +30,6 @@ public function getOfType()
         return Schema::resolveType($this->ofType);
     }
 
-    /**
-     * @return ObjectType|InterfaceType|UnionType|ScalarType|InputObjectType|EnumType|(Type&WrappingType)
```

Comment:
im not sure about this change 🤷🏾

Reply:
It is really not necessary, since any `Type` can be wrapped within a list. I guess we could clarify the return type of `NonNull::getWrappedType()`, which can not hold another `NonNull`.

Reply:
Ah right 👍🏾